chore(a11y): add documentation article for the new default ocean blue… (#1282)
telerik/blazor-docs: accessibility/accessibility-swatch.md

You can test the accessibility swatch in our [live demos](https://demos.telerik.com/blazor-ui/grid/overview) - it is initially selected in the `Change Theme` menu.

# Default Ocean Blue Accessibility Swatch

> The Default Ocean Blue Accessibility Swatch is built on top of [Ocean Blue](#ocean-blue-accessibility-swatch). The difference is that it has contrast focus indicators enabled and enforces compliance with WCAG requirements.

Reviewer: ```suggestion
The Default Ocean Blue Accessibility Swatch is built on top of [Ocean Blue](#ocean-blue-accessibility-swatch). The difference is that it has contrast focus indicators and is compliant with WCAG requirements.
```

Author: Done

> You can test the accessibility swatch in our [live demos](https://demos.telerik.com/blazor-ui/grid/overview) - select the `Default Ocean Blue` option in the `Change Theme` menu.

Reviewer: ```suggestion
You can test the accessibility swatch in our [live demos](https://demos.telerik.com/blazor-ui/grid/overview) - select the `Default Ocean Blue` option in the `Change Theme` dropdown menu.
```

> > **Limitations:** ColorPalette fails WCAG success criteria 1.4.11. Non-text contrast for the focus indicator on its items.

Reviewer: ```suggestion
> **Limitations:** ColorPalette fails WCAG success criteria 1.4.11. - "Non-text contrast for the focus indicator on its items".
```

Author: Done
